Gemfile.lock

```diff
@@ -60,6 +60,7 @@ GEM
       public_suffix (>= 2.0.2, < 5.0)
     autoprefixer-rails (10.0.1.0)
       execjs
+    aws_cf_signer (0.1.3)
```

Comment:
what is this one?

Reply:
These are the 2 gems I installed:
1- gem 'dotenv-rails'
2- gem 'cloudinary', '~> 1.16.0'

It must their doing in the gem.lock